requirements/test_functional.txt

```diff
@@ -0,0 +1,8 @@
+# sha256: bwGLPxbC1cSQ-u-cot0Pk4LpMPi3gDp1cVGMTDQE78o
+pytest==2.8.1
```

Comment:
`test_redirects.txt` uses `pytest 2.7.2` - should we be using the same version for consistency?